market/connector/jira/meta.json

```diff
@@ -0,0 +1,40 @@
+{
+	"id": "jira",
+	"name": "Jira",
+	"tags": ["connector"],
+	"listed": false,
```

Comment:
not visible on market yet ... still, the crucial bootstrap work for other business units to continue has been done :)

Reply:
jup. some further works needs to be done. e.g. Readme